stock-managment/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/MainContent.kt

```diff
@@ -187,7 +196,28 @@ fun MainContent(
         }
 
         Column(horizontalAlignment = Alignment.CenterHorizontally) {
-            Text(text = "Table content", textAlign = TextAlign.Center)
+            if (viewModel.toolbarTitle.collectAsState().value.name
+                == TransactionType.DISTRIBUTION.name
+            ) {
+                if (hasFacilitySelected && hasDestinationSelected == true) {
+                    MdcTheme {
```

Comment:
In compose, the theme has to be the top element in the tree.
`setContent {
  MdcTheme{
    MainContent()
  }
}
`